shared/haml/course_explorer_table.haml

```diff
@@ -136,6 +140,7 @@
   }
 
   courses << {
+    id: "csfe",
```

Comment:
Can we call this `exp` instead? I think it's too easily confused with CSF Course E.